testing/chain.go

```diff
@@ -261,6 +268,13 @@ func (chain *TestChain) QueryConsensusStateProof(clientID string) ([]byte, clien
 	return proofConsensus, consensusHeight
 }
 
+// GetSentPacket returns the sent packet with `sequence` (if any),
+// reconstructed from emitted events of type SendPacketEvent
+func (chain *TestChain) GetSentPacket(sequence uint64) (packet channeltypes.Packet, found bool) {
+	packet, found = chain.SentPackets[sequence]
+	return
+}
+
```

Comment:
Can we do without this? I don't think it's needed.

Reply:
I think it's useful to abstract away from the map data structure.

Reply:
Ehh, YAGNI